__mocks__/react-router-dom.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
See this file and how I'm using it in `Dashboard.test.js`

Because of this I had to add an `unmock` to a bunch of files that need the real `ReactRouter`, which unfortunately made this PR a little noisy. But the new pattern in `Dashboard.test.js` I think is much better for handling Links, and we should move over to that and ultimately be able to remove the `unmock`s.